src/cloud/aws/migration.md

```diff
@@ -42,12 +70,24 @@ We recommend using the AWS Systems Manager automation document to copy the insta
 - The instance ID of the source instance.
 - The subnet ID of the destination subnet.
 - The security group ID of the destination security group.
+- (Optionally) the name of the key pair defined for the previous VM for initial login
+
+Additional notes about using the AWS Systems Manager automation document:
+- If you use this automation, you will end up with two identical VM's running at the same time. 
+- Please consider what this will mean for any other systems depending on this VM. 
+- At a minimum, once you are satisfied that the copy is working correctly, you will need to shut the old VM down and eventually remove it. The automation does not clean up the old VM for you.
+- Pay attention to the region that is selected when running the automation document. It may not default to the correct region in which your resources are located. In particular, if the automation can't seem to find any of your resources, this is likely the cause.
 
 Once the automation document has been run, you will need to:
 - manually reassign any Elastic IP addresses from the source instance to the new instance, if applicable.
 - reassign any EC2 instance roles, if applicable.
 - clean up (delete/terminate) the source instance and any associated resources.
 - update any DNS records or other references to the source instance.
+- Potentially update the DNS servers within the VM. Note that the VM's internal DNS servers may be reset to use the default AWS provided DNS server. In particular, if using the campus shared subnet and you have overridden the default DNS servers to point to Infoblox, you will need to set this up again to enable proper DNS resolution for campus resources.
+
+If you were using Terraform to manage the old VM and want to continue using it to manage the new VM resource, you will need to define new resources and then import the copied VM into that. A way to get the correct syntax to use for importing is to run `terraform plan` and see how it defines the resource name. Once you have the name correct, if you were using a module for instance, import with something like `terraform import module.<module_name>.aws_instance.<resource_name> <instance_id>`. Once imported, run `terraform plan` again to see what it says will need to be changed. Observe what it would change, then update your Terraform code so unwanted changes would not occur. Keep doing this loop until the output of `terraform plan` reflects the actual changes (if any) that you want to make. Some common things that you may need to update inclued:
+- AMI - you will need to update to reflect the automatically created AMI from the automation
+- Key pair - set to "" if you didn't specify this when making the copy
```

Comment:
I'm sorry, but I don't see the value of complicating this document with more information. We're almost done with the migrations and no one that's left is using terraform, so this is really isn't for anyone. I am more in favor of a general terraform guidance document apart from this with tips and tricks like this.

Reply:
Ok, then do you have an alternate location that such content can be placed instead?